users/models/identity.py

```diff
@@ -120,12 +120,47 @@ def handle_moved(cls, instance: "Identity"):
 
     @classmethod
     def handle_deleted(cls, instance: "Identity"):
-        from activities.models import FanOut
+        from activities.models import (
+            FanOut,
+            Post,
+            PostInteraction,
+            PostInteractionStates,
+            PostStates,
+            TimelineEvent,
+        )
+        from users.models import Bookmark, Follow, FollowStates, HashtagFollow, Report
 
         if not instance.local:
             return cls.updated
 
+        # Delete local data including logins
```

Comment:
This says "including logins" but it does not seem to (and I would not want it to as that's on the User level) - is this an outdated comment?